packages/eds-core-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@equinor/eds-core-react",
-  "version": "0.17.0",
+  "version": "0.18.0-envis.3",
```

Comment:
I suppose this version number should go back to 0.17 before merging into develop

Reply:
Yes, good point!